<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tb_RecurringExpense">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>581</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="b_deleteExp">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>430</y>
      <width>84</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_editExp">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>390</y>
      <width>84</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_addExp">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>350</y>
      <width>84</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_totalSpend">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>581</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Spend:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>842</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
